frontend/assets/webpack-commons.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
                 rules: [
                     {
                         test: /\.(js|jsx)$/i,
+                        exclude: /(\.(spec|test)\.jsx?$)/,
```

Comment:
Maybe we can introduce this change in a follow up PR, because we may need to regenerate all bundled files to ensure it's all up to date. WDYT?